docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss

```diff
@@ -117,14 +53,14 @@
       height: hb-calculate-rems(6px);
       width: hb-calculate-rems(6px);
       border-radius: 50%;
-      background-color: $hb-color--black;
+      background-color: black;
```

Comment:
do we want this to remain hb-color--black? as a fallback? or an hb-global-color?

Reply:
Yeah, we'll want to get rid of that. I moved all the styles for both `ol` and `ul` into the `hb-colorful` mixin.